core/schema/src/main/liquibase/25.0.0/changelog.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-2.0.xsd" >
+    <changeSet author="mbrooks" id="25.0.0-kvstore_jsonb">
+        <preConditions onFail="MARK_RAN">
+            <not>
+                <tableExists tableName="kvstore_jsonb"/>
+            </not>
+        </preConditions>
+        <createTable tableName="kvstore_jsonb">
+            <column name="key" type="text">
+                <constraints nullable="false"/>
+            </column>
+            <column name="context" type="text">
+                <constraints nullable="false"/>
+            </column>
+            <column name="last_updated" type="timestamp">
+                <constraints nullable="false"/>
+            </column>
+            <column name="expires_at" type="timestamp">
+                <constraints nullable="true"/>
+            </column>
+            <column name="value" type="jsonb">
```

Comment:
What version of PostgreSQL was the `jsonb` type added in? Just wondering if this will change our PostgreSQL version requirements in any way.

Reply:
As we discussed in scrum, sounds like we will be requiring 10+ going forward after 25.0. Looks like jsonb was introduced in 9.4.